app/assets/javascripts/modules/autocomplete.js

```diff
@@ -0,0 +1,33 @@
+// Reinitializes autocomplete to add limit value. Fixes issue where
+// autocomplete results are hidden. Can remove when fixed upstream.
+
+$(document).ready(function() {
+  $('[data-autocomplete-enabled="true"]').each(function () {
+    var $el = $(this);
+
+    if ($el.hasClass('tt-hint')) {
+      return;
+    }
+
+    var suggestUrl = $el.data().autocompletePath;
+    var terms = new Bloodhound({
+      datumTokenizer: Bloodhound.tokenizers.obj.whitespace('value'),
+      queryTokenizer: Bloodhound.tokenizers.whitespace,
+      remote: {
+        url: suggestUrl + '?q=%QUERY',
+        wildcard: '%QUERY'
+      }
+    });
+    terms.initialize();
+    $el.typeahead({
+      hint: true,
+      highlight: true,
+      minLength: 2
+    }, {
+      name: 'terms',
+      limit: 10,
```

Comment:
Key here is the value 10, which is twice the number of returned results. Any less and the displayed results are progressively truncated to zero.

Reply:
I'm not certain of the status for our staging infrastructure at this moment, so I'm just trying to run this locally.